tests/unit/src/main/scala/tests/BaseAmmoniteSuite.scala

```diff
@@ -220,97 +220,95 @@ abstract class BaseAmmoniteSuite(scalaVersion: String)
     } yield ()
   }
 
-  if (scalaVersion != V.ammonite3)
```

Comment:
Note for the reviewers: the body of the test is not changed at all. I only removed the condition for excluding Scala 3